packages/govern-server/src/core/data/index.ts

```diff
@@ -0,0 +1,127 @@
+import { ErrorInvalidNetwork, ErrorUnexpectedResult } from '../errors'
+import { toNetwork } from '../utils'
+import { Address, Network, Networkish, QueryResult } from '../types'
+import * as queries from './queries'
+import TheGraphWrapper from './TheGraphWrapper'
+
+export type ConnectorTheGraphConfig = {
+  network: Networkish
+  subgraphUrl?: string
+  verbose?: boolean
+}
+
+function getSubgraphUrl(network: Network): string | null {
+  if (network.chainId === 1) {
+    return 'placeholder'
+  }
+  if (network.chainId === 4) {
+    return 'https://api.thegraph.com/subgraphs/name/0xgabi/aragon-govern-rinkeby-staging'
+  }
+  if (network.chainId === 100) {
+    return 'placeholder'
+  }
+  return null
+}
+
+class ConnectorTheGraph {
```

Comment:
I think the naming convention changed here. Shouldn't it be called ``TheGraphConnector`` and I even wouldn't it call ``Connector`` we could name it ``TheGraphRepository`` or ``TheGraphServiceClient`` this cause the ``Wrapper`` is the actual "connector" 🤷‍♂️

Reply:
I think this class could be the Govern JS API already? It would fetch data from The Graph (`TheGraphWrapper`), IPFS and other sources. We could move it one level up to be in `core/`, as it would be the main entry point to the core library.